intervals_2.html

```diff
@@ -918,12 +985,25 @@
     }
 
     function saveAsPdf() {
-      html2canvas(document.body, {
+      const pdfResultsContainer = document.createElement('div');
+      const container = document.getElementById('container');
+      const resultsContainer = document.getElementById('results-container');
+      container.appendChild(pdfResultsContainer);
+      pdfResultsContainer.appendChild(resultsContainer);
+      pdfResultsContainer.classList.add('pdf-results-container');
+
+      const generatePdfButton = document.querySelector('.generate-pdf-button');
+      generatePdfButton.classList.add('remove-button');
+      setTimeout( () => {
+        generatePdfButton.classList.remove('remove-button');
+        pdfResultsContainer.classList.remove('pdf-results-container');
+      }, 250);
+      html2canvas(document.querySelector('.pdf-results-container'), {
         onrendered: function(canvas) {
           const img = canvas.toDataURL("image/png");
           const doc = new jsPDF();
-          doc.addImage(img, 'JPEG', 0, 0, 180, 150);
-          doc.save('test.pdf');
+          doc.addImage(img, 'PNG', 0, 0, 180, 150);
```

Comment:
as discussed, let's see if we can fix the pdf problems by messing with these `addImage` args. PNG vs. JPEG isn't that important IMO

Reply:
Done. In addition, I made a mobile version of the viewbox because the .pdf was looking very stretched and contorted when made from smaller screen layouts.

Reply:
nice!!